user-managed-secret/variables.tf

```diff
@@ -0,0 +1,51 @@
+variable "admin_principals" {
+  description = "Principals allowed to peform admin actions (default: current account)"
+  type        = list(string)
+  default     = null
+}
+
+variable "description" {
+  description = "Description for this secret"
+  type        = string
+  default     = null
+}
+
+variable "environment_variables" {
```

Comment:
This output makes it possible to pass the module directly to https://github.com/thoughtbot/flightdeck/pull/121 as an input.